src/labs/requests/NewLabRequest.tsx

```diff
@@ -72,7 +72,9 @@ const NewLabRequest = () => {
       Toast(
         'success',
         t('states.success'),
-        `${t('lab.successfullyCreated')} ${newLab?.type} ${newLab?.patient}`,
+        `${t('labs.lab.successfullyCreated')} ${newLab?.type} ${t('labs.lab.for')} ${
+          newLab?.fullName
```

Comment:
`fullName` is not a property of Lab. `fullName` is a property of a `Patient`. We shouldn't update a model for this. We can look up the patient information by id.

Reply:
@jackcmeyer, understood. I will make this change shortly.